pom.xml

```diff
@@ -62,12 +62,13 @@ THE SOFTWARE.
   </scm>
 
   <properties>
-    <jenkins.version>1.609.3</jenkins.version>
-    <java.level>6</java.level>
+    <jenkins.version>2.60</jenkins.version>
```

Comment:
Needs feedback from the plugin maintainer. I am not comfortable with pushing master version and dropping support of all LTS versions just to get the improvement landed